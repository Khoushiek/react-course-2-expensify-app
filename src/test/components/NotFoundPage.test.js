import {shallow} from "enzyme";
import React from "react";
import NotFoundPage from "../../components/NotFoundPage";

test("should return Not found page component", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
})