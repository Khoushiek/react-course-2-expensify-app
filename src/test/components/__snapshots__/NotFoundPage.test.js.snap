// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return Not found page component 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Goto Home
  </Link>
</div>
`;
